app/javascript/packs/actions/buildings.js

```diff
@@ -34,6 +34,22 @@ export function removeBuilding(id, dispatch) {
   };
 }
 
+function saveBuilding(type, id, response) {
+  return {
+    type,
+    status: FETCH_SUCCESS,
+    buildingId: id,
+    response
+  };
+}
+
+function saveBuildingError(type, id, error) {
```

Comment:
I can rename this, but I'm not sure if we're using the buildings actions and reducers anywhere? This function has a call to saveError() which hasn't been defined

Reply:
Even if we're not, let's just set up the structure so that whoever picks this up will start off with the correct naming! Maybe make a `TODO` comment for implementing the `saveError` function or changing `saveError` within the `answers` reducer to a generic error handler function that accepts a `type` field and returns the action object (then it can be reused by other reducers).